README.md

```diff
@@ -5,29 +5,33 @@
 
 The Rust-ML working group is currently at an early stage of development, and is still deciding on several choices at an organizational level. This document will be updated accordingly as those decisions are made.
 
-### How can you contribute?
+## What do we do?
 
-Come say hello, read about, and weigh in on our [discussions](https://github.com/rust-ml/discussion)! We currently have several ongoing threads regarding upcoming scope of work, organizational structures, and more.
+In general, the Rust-ML working group's goals are to:
 
-The Rust-ML group has a Zulip channel. Feel free to take a look and say hi!
+- Create spaces where people interested in doing machine learning in Rust can meet, ask questions, and discuss their work
+- Help organize support for Rust libraries, tools, and projects in the machine learning domain
+- In rare instances, to assist with or assume the maintainership of projects important to the wider community
+
+We act as a resource to help people with experience in machine learning to become comfortable with Rust, enable people familiar with Rust to do machine learning, and provide guidance to people with neither. 
+
+## How can you contribute?
+
+The Rust-ML group has a Zulip chat. Feel free to take a look and say hi!
 - https://rust-ml.zulipchat.com
 
```

Comment:
or if you already written something ask for a code review! :)

Reply:
Good point! It probably makes sense to have that explicitly laid out